api/definitions/comment_rejection_object.md

```diff
@@ -0,0 +1,50 @@
+# Comment Rejection Object
+
+*Note: Info here isn't 100% complete, since we don't have sample information for what most comment rejection objects actually look like.*
+
+A comment rejection object looks like this:
+
+```
+{
+    rejected: /* Rejection code - see below */
+    status: /* (optional) Status object with rejection details - see below */
+    appealId: /* (if IP muted) Case number to contact ST with to appeal block */
+}
+```
+
+The `rejected` property will be one of the following:
+
+- `"isEmpty"`: the comment is empty
+- `"isFlood"`: sending too many comments in a short period of time
+- `"isBad"`: content was caught by the filter bot
+- `"hasChatSite"`: content contained a link to a website with unmoderated chat
+- `"isSpam"`: same content has been posted too many times
+- `"isMuted"`: posted too many bad comments and account has been muted for the day
+- `"isUnconstructive"`: content was detected as being mean or disrespectful
+- `"isDisallowed"`: tried to post on a page where comments are disabled
+- `"isIPMuted"`: comment sent from a network blocked for posting the community guidelines too many times
```

Comment:
Looks like a typo - did you mean "breaking the community guidelines"?  I doubt posting the CG's too many times will get you banned 😄

Reply:
Lol, thanks! Will fix.